about.html

```diff
@@ -201,46 +228,59 @@ <h2>Company Overview</h2>
         <h2>Mission and Vision</h2>
         <div class="section-img">
           <img
-            src="https://imgs.search.brave.com/2CAFCbM6U5I6Q03dK-bC8qqqyb_mjrDxo4YGu4-zCnk/rs:fit:500:0:0/g:ce/aHR0cHM6Ly9pbWFn/ZXMuZnJlZWltYWdl/cy5jb20vaW1hZ2Vz/L2xhcmdlLXByZXZp/ZXdzLzMxOC9pbi10/aGUtY2FyLTEtMTQ5/NTA3MC5qcGc_Zm10"
-            alt="Mission and Vision Image"
+            src="https://imgs.search.brave.com/q0e8cW8B-XzUbrgZx4qP0ObqkV3g5cPE82Dptj32vVY/rs:fit:860:0:0/g:ce/aHR0cHM6Ly9pbWcu/ZnJlZXBpay5jb20v/ZnJlZS1waG90by9w/aG90by1jb250ZW50/LzIwMTkvMDMvbXNo/aXN0LzE1NTM5MDA2/bmF0dXJlcmVpbC9z/bGFkZXJzL3ZhbC9j/YWxlbmRhci5qcGc"
+            alt="Mission Vision Image"
           />
         </div>
         <p>
-          <strong>Mission:</strong> To provide reliable and affordable cab
-          rental services while ensuring exceptional customer experience.
-        </p>
-        <p>
-          <strong>Vision:</strong> To be the leading cab rental company known
-          for innovation, sustainability, and outstanding service.
+          Our mission is to provide safe, efficient, and affordable transportation
+          services to all. We envision a future where everyone has access to
+          reliable transportation, enhancing their mobility and quality of life.
         </p>
       </section>
 
       <!-- Values and Principles -->
       <section id="values-principles" class="values-principles">
         <h2>Values and Principles</h2>
         <ul>
-          <li>
-            <strong>Customer First:</strong> We prioritize our customers' needs
-            and strive to exceed their expectations.
-          </li>
-          <li>
-            <strong>Integrity:</strong> We operate with honesty and transparency
-            in all our dealings.
-          </li>
-          <li>
-            <strong>Safety:</strong> We ensure the highest safety standards for
-            our customers and drivers.
-          </li>
-          <li>
-            <strong>Innovation:</strong> We continuously seek innovative
-            solutions to enhance our services.
-          </li>
-          <li>
-            <strong>Sustainability:</strong> We are committed to sustainable
-            practices and reducing our environmental impact.
-          </li>
+          <li>Customer Satisfaction</li>

```

Comment:
Could you share the thoughts behind changing the content ? 

Also after changing content there's a lot of white space that it feels empty. 

Suggestions: 
1. Experiment with text alignment.
2. Experiment with the box sizing if we even need that big values and principles boxes. 

<img width="1232" alt="Screenshot 2024-10-07 at 00 43 54" src="https://github.com/user-attachments/assets/f2a03a0d-7571-486c-9f8a-da022d025556">